src/verifier/tests.rs

```diff
@@ -224,3 +299,22 @@ pub fn test_type_errors() {
         assert!(!&res);
     }
 }
+
+//WE DONT HAVE BEGIN WORKING YET
```

Comment:
Add a `TODO: uncomment after BEGIN is working` to this comment so it's flagged in the IDEs as something that needs to be done.